Source/Analytics/Read/DataCollectors/District.cs

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) The International Federation of Red Cross and Red Crescent Societies. All rights reserved.
+*  Licensed under the MIT License. See LICENSE in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+using Concepts;
+using Dolittle.ReadModels;
+
+namespace Read.DataCollectors
+{
+    public class District : IReadModel
+    {
+        public DistrictId Id { get; set; }
+        public RegionName RegionName { get; set; }
```

Comment:
I know we discussed this but it feels a bit iffy that we reference a region by its name when we have defined an (certain) unique id for it.